pty/pty_other.go

```diff
@@ -56,6 +56,7 @@ func (p *otherPty) Close() error {
 
 	err := p.pty.Close()
 	if err != nil {
+		_ = p.tty.Close()
```

Comment:
How do we know if there would be no error this time? We’re only trying this twice, right?

Reply:
I wonder if we should return the newer error instead in that case.

Reply:
Usually the first error encountered is returned, in this case pty and tty are different resources, so I can't think of a reason for prioritizing one over the other.

Reply:
Aha, I misread! Thanks for clarifying.